content/10.diagnostics.md

```diff
@@ -214,24 +214,37 @@ This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG
 
-Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036].
-One study assessed sustained immunity using 245 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063]. 
+Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].
+Dan et al. assessed sustained immunity using 245 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063]. 
 The samples were collected at various time points between 6 and 240 days post-symptom onset, meaning some patients were assessed longitudinally. 
 Of the samples, 43 were collected at least 6 months after symptom onset.
 After 1 month, 98% of patients were seropositive for IgG to the spike protein, S. 
 Moreover, S IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
 Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
 
-The development of memory B cells and memory T cells has also been measured.
-SARS-CoV-2 spike-specific memory B cell levels steadily rose over the first 120 days following symptom onset [@doi:10.1126/science.abf4063].
-RBD-specific memory B cells had been detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @pmid:33443036], and this study confirms these finding and shows that these cells steadily increase over the 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
-Nucleocapsid-specific memory B cells seem to follow a similar pattern [@doi:10.1126/science.abf4063].
+The findings of Dan et al. are in accordance with Sherina et al. who examined 119 samples from 88 donors who had mild to severe COVID-19 [@doi:10.1016/j.medj.2021.02.001]. 
+They observed a relatively stable level of IgG and plasma neutralizing antibodies up to 6 months post diagnosis.
+Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6-8 months post symptom-onset.
+A study by Gaebler et al. has shown that titers of IgM and IgG antibodies against the RBD decrease from 1.3 to 6.2 months post infection in 87 individuals [@doi:10.1038/s41586-021-03207-w]. 
+IgM seemed to show the greatest decrease of anti-RBD reactivity (53%), followed by IgG (32%), but a lower decline of IgA (15%) activity was observed. 
+It was noted that higher levels of anti-RBD IgG and anti-N total antibodies were detected in individuals that reported persistent post-acute symptoms at both study visits.
```

Comment:
I assumed it might have been earlier when discussing antibody diagnostics. Only way to confirm whether you had infection prior to receiving a vaccine. Because the vaccine induces antibodies for spike. Super cool